CONTRIBUTING.md

```diff
@@ -40,33 +40,17 @@ brm generate
 You should be able to see these files created in the module folder:
 | File Name | Description |
 | :--------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `metadata.json` | An JSON file containing module metadata. You must edit the file to provide the metadata values. |
-| `main.bicep` | An empty Bicep file that you need to update. This is the main module file. |
+| `main.bicep` | An empty Bicep file that you need to update. This is the main module file. You must edit the "metadata" statements to provide the module name, description and owner. |
 | `test/main.test.bicep` | A Bicep file to be deployed in the PR merge validation pipeline to test if `main.bicep` is deployable. You must add at least one test to the file. A module referencing `main.bicep` is considered a test. |
 | `main.json` | The main ARM template file compiled from `main.bicep`. This is the artifact that will be published to the Bicep public registry. You should not modify the file. |
-| `README.md` | The README file generated based on the contents of `metadata.json` and `main.bicep`. You need to update this file to add examples. |
+| `README.md` | The README file generated based on the contents of `main.bicep`. You need to update this file to add examples. |
 | `version.json` | The module version file. It is used together with `main.json` to calculate the patch version number of the module. Every time `main.json` is changed, the patch version number gets bumped. The full version (`<ModuleMajorVersion>.<ModuleMinorVersion>.<ModulePatchVersion>`) will then be assigned to the module before it gets published to the Bicep public module registry. The process is handled by the module publishing CI automatically. You should not edit this file. |
 
-### Authoring module files
-
-The files that you need to edit are `metadata.json`, `main.bicep`, `test/main.test.bicep`, `README.md`, and `version.json`.
+NOTE: The `metadata.json` file is now obsolete. Running `brm generate` will automatically remove this file and place its contents into the main.bicep file as `metadata` statements.
```

Comment:
If we wanted to be really verbose - could show an example of the metadata statements.

Reply:
Added "(see below)"

8aaeda01828a327b4d2fbe4ab31b9b6785efd6fd